modules/ROOT/pages/to-find-info.adoc

```diff
@@ -19,14 +19,20 @@ the HTTP functions in the REST specification.
 
 == Exchange Search
 
-Search lets you find Exchange assets that contain one or more terms
-in an asset title, asset ID, or a tag in an asset. You can also combine search terms with the filters listed in All Types.
-Searches only apply to the Exchange you are viewing.
-If you search in a private Exchange, the search only lists the
-private Exchange assets and not public Exchange assets.
+Exchange Search looks for terms in asset names, tags, categories, properties defined in API specifications, custom attributes (also known as custom fields), and the portal content shown on the asset page. Search includes RAML and OAS APIs and all other assets in Exchange.
+
+Assets migrated from Exchange 1 also allow the search to include text in an asset's description.
+
+You can combine search terms with the filters listed in All Types.
 
 You can also search by clicking Category or Tag elements. This creates a filtered search you can use to find similar assets.
 
+Searches only apply to the Exchange you are viewing.
+
+If you search in a private Exchange, the search only lists the private Exchange assets and not public Exchange assets.
+
+If a search matches some versions of an asset and not other versions of the asset, then it returns the asset's latest matching version, even if that is not the asset's latest version.
```

Comment:
This is kind of confusing. @LucasLudueno @mcampo What do you guys think?

Reply:
Would this be better? "A search may match some versions of an asset and not others. In this case Exchange returns the latest version that matches the search. This may not be the latest version."

Reply:
This is not true **yet**. We still need to change the listings page to make asset cards navigate to a specific version. Let's remove this sentence.

Reply:
I like that last phrasing better